airbyte-integrations/connectors/source-shopify/source_shopify/schemas/customer_journey_summary.json

```diff
@@ -191,6 +191,82 @@
               "type": ["null", "string"]
             }
           }
+        },
+        "moments": {
+          "description": "The events preceding a customer's order, such as shop sessions.",
+          "type": ["null", "array"],
+          "items": {
+            "type": "object",
```

Comment:
nit: ["null", "object"]

Reply:
added updates. what are the next steps for the CI approval? do you have the ability to kick them off ?